packages/cdktf-cli/bin/cmds/handlers.ts

```diff
@@ -197,6 +197,13 @@ export async function init(argv: any) {
   await displayVersionMessage();
   await checkEnvironment();
 
+  console.log(argv);
+  if (argv.fromTerraformProject === "") {
```

Comment:
Wouldn't this still fail when the user passes `.`?

Reply:
yeah, do you think this is a case we should check? `--from-terraform-project=$(pwd)` would also fail 🤔

Reply:
Yeah, I'd check that too and adjust the error message

Reply:
(the latter you already did)